.buildbot.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# vim: expandtab sts=0 sw=4 smarttab
+set -e
+
+echo "Test buildbot" 
+
```

Comment:
Try to do a simple `clang-format` check so we are sure that the communication with `unittest` is right.

Reply:
I don't think we can do clang-format on Boehm: we need to stay in sync with upstream.

However, @djichthys I think we should at least compile Boehm in buildbot, even if we don't run any tests (although running tests would be good too!).

Reply:
That's right. I had completely forgotten about upstream.